sesman/chansrv/smartcard_pcsc.c

```diff
@@ -1650,7 +1652,8 @@ scard_process_cmd_version(struct trans *con, struct stream *in_s)
     LOG_DEVEL(LOG_LEVEL_DEBUG, "scard_process_version: major %d minor %d", major, minor);
```

Comment:
Please replace this with `LOG(LOG_LEVEL_INFO` so we get useful error reports from users!

Reply:
Thanks, modified